Apps/APN-UIKit/APN UIKit/View/DashboardViewController.swift

```diff
@@ -28,6 +31,26 @@ class DashboardViewController: BaseViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        // In a future PR, we will remove the asyncAfter(). this is only for testing in sample apps because when app opens, the local queue is empty. so wait to check messages until first fetch is done.
+        DispatchQueue.main.asyncAfter(deadline: .now() + 5) { [self] in
+            inlineInAppView.elementId = "dashboard-announcement"
+
+            // We want to test that Inline Views can be used by customers who prefer to use code to make the UI.
+            // Construct a new instance of the View, add it to the ViewController, then set constraints to make it visible.
+            let newInlineViewUsingUIAsCode = InAppMessageView(elementId: "dashboard-announcement-code")
+            // Add the View to the screen.
+            view.addSubview(newInlineViewUsingUIAsCode)
+
+            // We are adding the new View between 2 Buttons in the StackView. No specific reason why, not sure where else to put it.
+            newInlineViewUsingUIAsCode.widthAnchor.constraint(equalToConstant: UIScreen.main.bounds.width).isActive = true
+            newInlineViewUsingUIAsCode.topAnchor.constraint(equalTo: randomEventButton.bottomAnchor).isActive = true
+            newInlineViewUsingUIAsCode.bottomAnchor.constraint(equalTo: customEventButton.topAnchor).isActive = true
+            newInlineViewUsingUIAsCode.leadingAnchor.constraint(equalTo: view.leadingAnchor).isActive = true
+            newInlineViewUsingUIAsCode.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
+            newInlineViewUsingUIAsCode.centerXAnchor.constraint(equalTo: randomEventButton.centerXAnchor).isActive = true
```

Comment:
During QA testing, I noticed what might be a bug with this code. 

steps: 
* Send yourself 2 inline test messages so you have both inline Views showing messages in the app. 
* Scroll down the screen. When you put your finger in the middle of 1 of the inline Views and scroll your finger, the screen moves with your finger. But for the other inline View (this one I added via code), the screen does not move with your finger. 

I think that this is a bug with the AutoLayout code here which means it's a bug in the sample app and not with the SDK.